### PR TITLE
home page created

### PR DESCRIPTION
**Pull request by anandukch**

### CODE REVIEW

**Comment by UKnowWhoIm** [comment]
- Make meaningful commit messages
- Regarding video quality, the original video resolution is low(960x402), maybe due to compression by whatsapp?
- There was a page before the video right?
- Mute button :smile: 

@anandukch

**Comment by anandukch** [comment]
- Yes Sure😅😅
- Yes, it mIght be the reason
- Which page??
- 😎😎😎😎

**Comment by irenekurien** [comment]
>     * There was a page before the video right?
> 
> @UKnowWhoIm @anandukch Not a page, it's like an overlay for some sec till the video loads.

**Comment by UKnowWhoIm** [comment]
That "mute button" animation makes sense for loading screens, not for mute/unmute, I think we could find use for that during loading.

For this case, try https://fontawesome.com/v6.0/icons?q=volume&s=solid%2Cbrands buttons, we'll some lose immersion, but I think this is the best way forward.

And where is the change @irenekurien suggested about the splash screen

@anandukch

**Comment by irenekurien** [comment]
@anandukch Replace the video with the original (non compressed) one

**Comment by anandukch** [comment]
original video got lost from adarsh ser @irenekurien

File: js/script.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
+var splash = document.querySelector(".splash-screen");
```
Comment (irenekurien) [inline js/script.js]: Don't use var, replace it with let or const accordingly

**Review by irenekurien — CHANGES_REQUESTED** [review]

**Comment by irenekurien** [comment]
> original video got lost from adarsh ser @irenekurien

Great 🌔

**Review by irenekurien — APPROVED** [review]

**Pull request merged**